### ISSUE TITLE
Python project: SentimentScanner 

### ISSUE THREAD

**Issue by tuhinaww** [debrajrout/Dev.HFest#88]
Sentiment Analysis with the help of Bert model

**Comment by debrajrout** [comment]
> Hey, I would like to add on a python project!
> Sentiment Analysis with the help of Bert model
> It will be made using both ML as well as the frontend integration using, flask app
> 
> Thanks & Regards,
> Tuhina

Yes you can. But you have to maintain the proper folder structure

**Comment by tuhinaww** [comment]
@debrajrout ,Sure thing!

**Comment by tuhinaww** [comment]
Created a PR for the same, please check @debrajrout
